### DISCUSSION TITLE
Longer sync times for LFOs

### DISCUSSION THREAD

**Discussion by BM-01** [SynthstromAudible/DelugeFirmware#713] [Ideas]
Hey really appreciate all the hard work everyone is putting into this!

Is there any chance we could get sync modes for 4, 8 and 16 bars? I find the 2 bars is too short for certain things like using an LFO to slowly control panning etc.

Thanks,
Brad

**Comment by thejesiah** [comment]
I really, really, really want this. You can do it in non-sync mode, often even multiples work like "1", "2", "5", "10"..but it's a guessing game, really. 16 or even 64 bars would be great. Sometimes you just want that filter to open up slowly for half a song or more!

**Comment by Ponchomouse** [comment]
Came here to say this. Drawn automation kind of works for this but an 8 bar lfo would be amazing.

**Comment by nikodemus** [comment]
You can do this already. Settings -> Defaults -> Resolution = 96.

Gives you LFO sync range from 8 bars to 32nd notes.

The changed resolution only applies to new songs

**Reply by Ponchomouse** [reply]
Ah great, this works really well. Many thanks for sharing. I guess the downside is losing resolution as looks like 32nd notes are the smallest interval at that resolution. Be great to get longer lfos at higher resolutions at some point but this works well.

**Comment by worker-gray** [comment]
It would be nice if both LFO's could be clock sync'd without the need for changing resolution.
Alternately, it would be helpful if there was a way to retroactively change the song resolution without having to reformat the song.
Clock sync'd envelopes that could repeat would be sweet.

**Comment by nikodemus** [comment]
As it happens, I have LFO2 sync support in the works: https://github.com/SynthstromAudible/DelugeFirmware/pull/2005

**Reply by worker-gray** [reply]
AWESOME!

**Reply by nikodemus** [reply]
LFO2 sync has been merged and is available on the nightly build.
